pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -216,30 +273,36 @@ func (r *reconciler) add(rule *CompletedRule) error {
 				From:      from,
 				To:        podsToOFAddresses(pods),
 				Service:   filterUnresolvablePort(servicesMap[svcHash]),
+				Action:    rule.Action,
+				Priority:  ofPriority,
 			}
 		}
 
 		// If there are no "ToAddresses", the above process doesn't create any PolicyRule.
 		// We must ensure there is at least one PolicyRule, otherwise the Pods won't be
 		// isolated, so we create a PolicyRule with the original services if it doesn't exist.
 		// If there are IPBlocks or Pods that cannot resolve any named port, they will share
-		// this PolicyRule.
-		svcHash := hashServices(rule.Services)
-		ofRule, exists := ofRuleByServicesMap[svcHash]
-		// Create a new Openflow rule if the group doesn't exist.
-		if !exists {
-			ofRule = &types.PolicyRule{
-				Direction: v1beta1.DirectionOut,
-				From:      from,
-				To:        []types.Address{},
-				Service:   filterUnresolvablePort(rule.Services),
+		// this PolicyRule. ClusterNetworkPolicy do not need this default isolation.
+		if !rule.isAntreaNetworkPolicyRule() {
```

Comment:
maybe
if rule.isAntreaNetworkPolicyRule() {
    return nil
}

so you don't have to do indentation

Reply:
We cannot return here. Need to do installOFRule later in the function.